src/common/Container/styled.js

```diff
@@ -2,13 +2,10 @@ import styled from "styled-components";
 
 export const Wrapper = styled.div`
     max-width: 1408px;
-    padding: 56px 20px 100px;
+    padding: 0px 20px;
```

Comment:
We need top and bottom padding for Container because on list pages there is no space between Title and Header and Pagination and the bottom of the page.

Reply:
This is a mistake, thanks!

Reply:
Цe will discuss it ;)